src/grocker/builders/op.py

```diff
@@ -98,7 +100,11 @@ def get_manifest_digest(name):
         return None  # Docker HUB API is not documented
 
     registry, repository = registry_repository.split('/', 1)
-    response = requests.head('https://{}/v2/{}/manifests/{}'.format(registry, repository, tag))
+    auth_info = None
+    with open('.docker/config.json', 'r') as config_file:
```

Comment:
I'm not sure this solution is general enough: the file might not exist or exist in an other place depending on the OS, installation, etc. Let's forget this part and simply keep the manifest option.